### PR TITLE
Configure burn staging envs

### PR DESCRIPTION
**Pull request by sunny-viktoryia**
partially closes https://github.com/sparkletown/internal-sparkle-issues/issues/804
partially closes: https://github.com/sparkletown/internal-sparkle-issues/issues/780

### CODE REVIEW

**Review by yurii-lubynets — APPROVED** [review]

File: .firebaserc
Comment (mike-lvov) [inline .firebaserc:24]: I wonder if you can leave it as `sparkle-demo` here. Not sure about all of the consequences though.

**Review by mike-lvov — APPROVED** [review]
LGTM to merge in. Maybe @0xdevalias can have suggestions on how to map sparkle-demo-e65b1 to sparkle-demo

**Comment by mike-lvov** [comment]
@0xdevalias turned off `codeowners` to facilitate the creation of `sparkle-burn-staging` to unblock our outsource partners

**Pull request merged**
